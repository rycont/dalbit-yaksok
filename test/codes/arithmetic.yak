1 + 1 보여주기

1 + 2 * 3 보여주기

5 - 3 / 2 보여주기

내_나이 = 10
내_나이 보여주기
내_나이 += 10
내_나이 보여주기
내_나이-=3
내_나이 보여주기
내_나이%= 8
내_나이 보여주기
내_나이*=128
내_나이 보여주기
내_나이 /=8
내_나이 보여주기
