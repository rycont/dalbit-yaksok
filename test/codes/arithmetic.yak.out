2
7
3.5
10
20
17
1
128
16
